src/pages/workspace/reports/WorkspaceReportsPage.tsx

```diff
@@ -80,15 +79,14 @@ function WorkspaceReportFieldsPage({
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
         return Object.fromEntries(Object.entries(policy.fieldList).filter(([_, value]) => value.fieldID !== 'text_title'));
     }, [policy]);
-    const hasAccountingConnection = !isEmptyObject(policy?.connections);
     const [isOrganizeWarningModalOpen, setIsOrganizeWarningModalOpen] = useState(false);
 
     const onDisabledOrganizeSwitchPress = useCallback(() => {
-        if (!hasAccountingConnection) {
+        if (!hasReportAccountingConnections) {
```

Comment:
```suggestion
        if (!hasAccountingConnections) {
```

Rename this variable please